corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -652,3 +652,8 @@ def render(self, context):
     nodelist = NodeList([FakeNode()])
 
     return AddToBlockNode(nodelist, 'initial_page_data')
+
+
+@register.filter
+def concat_list(str_elements):
+    return ", ".join(str_elements)
```

Comment:
this might be useful: https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#join